### PR TITLE
Create Register Completed Screen

### LINKED ISSUE #14: Add registration completed screen design
# REQUIREMENTS
When the user gave his data required, it will show a screen that says "Registro completado", a welcome message, and the "Iniciar" button. When the user touch that button, his session in the "covid-tracker app" has begun  

# DESIGN
image

# Acceptance Criteria
List all required acceptance criteria
- [ ] Implementation must match the design provided.
- [ ] The navigation between screen can move until this new screen
- [ ] show a test screen when the "Iniciar" button has been touched

### PR DESCRIPTION
**Pull request by xErikPsx**
# Description

- New Register Completed Screen.
- There's no functionallity in "Iniciar" button until the incoming issues.

#Shout out
Thanks to @OwenRamire who helped to solve this issue.
 
  ## Screenshots 

image

closes

### CODE REVIEW

File: src/screens/RegisterCompleted.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  mainText: {
+    flex: 2,
+  },
+
+  title: {
+    fontSize: 27,
+    textAlign: 'center',
+    flex: 0.5,
+  },
+  succesContainer: {
```
Comment (OwenRamire) [inline src/screens/RegisterCompleted.js:74]: Add the "s" on the `succesContainer` (`successContainer`) property and also in the component where you use this property

File: src/screens/Welcome.js [hunk 2]
```diff
@@ -32,7 +32,7 @@ const Welcome = ({navigation}) => {
         <TouchableOpacity
           style={styles.button}
           onPress={() => {
-            console.log('Hello World');
+            navigation.navigate('RegisterCompleted');
```
Comment (OwenRamire) [inline src/screens/Welcome.js]: I assume that when you navigate into the `RegisterCompleted` screen, you navigate when the  user put his contact information like this screen not when the user touches the "Iniciar sesión" button 

image

File: src/screens/RegisterCompleted.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+};
+
+export default RegisterCompleted;
+
+const styles = StyleSheet.create({
+  mainContainer: {
+    flex: 1,
+    backgroundColor: Colors.white,
+  },
+
```
Comment (OwenRamire) [inline src/screens/RegisterCompleted.js]: Delete all the white spaces that you have

File: src/screens/RegisterCompleted.js [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+        <View style={styles.mainText}>
+          <Text style={styles.title}>Tu cuenta se ha creado</Text>
+          <View style={styles.textContainer}>
+            <Text style={styles.paragraphs}>
+              Gracias por registrarte para protegerte y a proteger a tu
+              comunidad.
+            </Text>
+            <Text style={styles.paragraphs}>
+              Juntos podemos detener la propagación del COVID-19
+            </Text>
```
Comment (OwenRamire) [inline src/screens/RegisterCompleted.js]: Put the text in a constant message on the file `src/utils/constants` and import them

**Review by OwenRamire — CHANGES_REQUESTED** [review]

File: src/screens/RegisterCompleted.js [hunk 5]
```diff
@@ -20,14 +21,13 @@ const RegisterCompleted = ({navigation}) => {
       </View>
       <View style={styles.succesContainer}>
```
Comment (OwenRamire) [inline src/screens/RegisterCompleted.js:22]: Change `styles.succesContainer` for `styles.successContainer`

**Review by OwenRamire — CHANGES_REQUESTED** [review]

**Review by OwenRamire — DISMISSED** [review]
🚀

File: src/screens/RegisterCompleted.js [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+    textAlign: 'center',
+    flex: 0.5,
+  },
+  succesContainer: {
+    justifyContent: 'space-between',
+    paddingHorizontal: 20,
+    paddingVertical: 20,
+    flex: 2,
+  },
+
```
Comment (luisarmandodv) [inline src/screens/RegisterCompleted.js]:
```suggestion
```

File: src/screens/RegisterCompleted.js [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+  heartHandsImage: {
+    width: 200,
+    height: 200,
+    resizeMode: 'contain',
+  },
+
+  mainText: {
+    flex: 2,
+  },
+
```
Comment (luisarmandodv) [inline src/screens/RegisterCompleted.js]:
```suggestion
```

File: src/screens/RegisterCompleted.js [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+    alignItems: 'center',
+    paddingVertical: 20,
+    marginTop: 15,
+  },
+  heartHandsImage: {
+    width: 200,
+    height: 200,
+    resizeMode: 'contain',
+  },
+
```
Comment (luisarmandodv) [inline src/screens/RegisterCompleted.js]:
```suggestion
```

File: src/screens/RegisterCompleted.js [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+              {REGISTER_COMPLETED_TEXT.together}
+            </Text>
+          </View>
+        </View>
+        <TouchableOpacity
+          style={styles.btnAccept}
+          onPress={() => {
+            console.log('Register Completed');
+          }}>
+          <Text style={styles.textBtnAccept}>Iniciar</Text>
```
Comment (luisarmandodv) [inline src/screens/RegisterCompleted.js]: I see you already have a constant file for text. Could you add this text to that file?

**Review by luisarmandodv — CHANGES_REQUESTED** [review]

**Review by luisarmandodv — APPROVED** [review]

**Review by OwenRamire — APPROVED** [review]

**Pull request merged**